core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow.component.spec.ts

```diff
@@ -28,26 +28,19 @@ import { NzDatePickerModule } from "ng-zorro-antd/date-picker";
 import { en_US, NZ_I18N } from "ng-zorro-antd/i18n";
 import { NoopAnimationsModule } from "@angular/platform-browser/animations";
 import { OperatorMetadataService } from "src/app/workspace/service/operator-metadata/operator-metadata.service";
-import { mockOperatorMetaData } from "src/app/workspace/service/operator-metadata/mock-operator-metadata.data";
-import { AppSettings } from "src/app/common/app-setting";
 import { StubOperatorMetadataService } from "src/app/workspace/service/operator-metadata/stub-operator-metadata.service";
 import { NzUploadModule } from "ng-zorro-antd/upload";
-import { MatSelectModule } from "@angular/material/select";
-import { By, EVENT_MANAGER_PLUGINS } from "@angular/platform-browser";
-import { animationFrameScheduler } from "rxjs";
+import { By } from "@angular/platform-browser";
 import { ScrollingModule } from "@angular/cdk/scrolling";
 import { NzAvatarModule } from "ng-zorro-antd/avatar";
 import { NzToolTipModule } from "ng-zorro-antd/tooltip";
-import { FileSaverService } from "src/app/dashboard/service/user-file/file-saver.service";
+import { FileSaverService } from "../../service/user-file/file-saver.service";
 
 describe("SavedWorkflowSectionComponent", () => {
```

Comment:
The `.spec` files repeat the class name here and should be changed too.
Similar issue in other files as well.

Reply:
Noted. Will fix later.